tests/headers/no-default-opaque.hpp

```diff
@@ -0,0 +1,7 @@
+// bindgen-flags: --no-default "NoDefault" --opaque-type "NoDefault"
+
+// `--with-derive-default` is added by the test runner implicitly.
```

Comment:
Does it complain if you add the flag anyways? If not I'd just do that.

Reply:
Unfortunately the duplicate flag causes an error since the "with-derive-default" flag was not defined with multiple=true, so I can't add it again.